lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -121,9 +121,8 @@ struct CommitState
     bool Empty()
     {
         return _indexBufferData == nullptr && _shader == nullptr && _enabled == nullptr
-            && !_geometryDirty && _boundingBox == nullptr && !_renderItemData._usingInstancedDraw
-            && _instanceTransforms.length() == 0 && _ufeIdentifiers.length() == 0
-            && _worldMatrix == nullptr;
+            && !_geometryDirty && _boundingBox == nullptr && !_instanceTransforms
+            && !_instanceColors && _ufeIdentifiers.length() == 0 && _worldMatrix == nullptr;
```

Comment:
Remove the _usingInstancedDraw check, because we no longer always update instancing.